src/ui/Views/Courses/Shared/RequestAccessSuccess.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@if (TempData["RequestAccess_To_Name"] != null) {
```

Comment:
I would like to see a strongly typed wrapper around TempData that represents our usages.
Something like https://stackoverflow.com/a/10956414/10245 perhaps